From 2e17c0fb48d8e051c24b5b7c413dc722c318bced Mon Sep 17 00:00:00 2001
From: Nicolas LOUBOUTIN <nicolas.louboutin@st.com>
Date: Thu, 12 Nov 2020 12:05:54 +0100
Subject: [PATCH] Android: allow file system access

Keep the possibility for debug purpose to access the file system (kernel_write).
Target: dump to file

Signed-off-by: Nicolas LOUBOUTIN <nicolas.louboutin@st.com>
Change-Id: I23c6bf05d9a9ba901128844c89cce2230d7fc9ca
---
 hal/os/linux/kernel/gc_hal_kernel_driver.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hal/os/linux/kernel/gc_hal_kernel_driver.c b/hal/os/linux/kernel/gc_hal_kernel_driver.c
index 9e0c2ea..2b59284 100644
--- a/hal/os/linux/kernel/gc_hal_kernel_driver.c
+++ b/hal/os/linux/kernel/gc_hal_kernel_driver.c
@@ -69,7 +69,7 @@
 
 MODULE_DESCRIPTION("Vivante Graphics Driver");
 MODULE_LICENSE("Dual MIT/GPL");
-
+MODULE_IMPORT_NS(VFS_internal_I_am_really_a_filesystem_and_am_NOT_a_driver);
 
 static struct class* gpuClass = NULL;
 
-- 
2.17.1

